src/components/Map/MarkerList.js

```diff
@@ -620,15 +626,9 @@ const mapStateToProps = (state) => ({
   activeMap: state.mapUtils.activeMap,
   pathLink: state.stopPlace.pathLink,
   showExpiredStops: state.stopPlace.showExpiredStops,
-  disabled:
-    (state.stopPlace.current &&
-      state.stopPlace.current.permanentlyTerminated) ||
-    !getIn(state.roles, ["allowanceInfo", "canEdit"], false),
-  disabledForSearch: !getIn(
-    state.roles,
-    ["allowanceInfoSearchResult", "canEdit"],
-    false,
-  ),
+  disabled: !getStopPermissions(state.stopPlace.current).canEdit,
+  stopPlace: state.stopPlace.current,
+  disabledForSearch: !getStopPermissions(state.stopPlace.current).canEdit,
```

Comment:
This change is dubious but not sure what "disabledForSearch" actually means. Needs investigation.